### PR TITLE
SAM SCORE IMPROVEMENT FINAL

### PR DESCRIPTION
**Pull request by vibhuti910**
Improved the SAM score of frontend from 4.3 to 4.84
Improved the SAM score of the backend from 4.4  to 4.64

### CODE REVIEW

File: src/dagent/postman/D-Agent.postman_environment.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "values": [
     {
       "key": "host",
-      "value": "localhost:3000",
+      "value": "107.99.41.212:3000",
```
Comment (manoj22-k) [inline src/dagent/postman/D-Agent.postman_environment.json:7]: please use localhost

File: src/mtool/ui/src/containers/Dashboard/index.test.js [hunk 2]
```diff
@@ -110,7 +110,47 @@ describe("Dashboard", () => {
       </Router>
     );
   };
+ // duplicate code 
 
+ const configure_json = {
```
Comment (manoj22-k) [inline src/mtool/ui/src/containers/Dashboard/index.test.js:115]: please use camelCase for variables in js files

File: src/mtool/ui/src/containers/Dashboard/index.test.js [hunk 3]
```diff
@@ -110,7 +110,47 @@ describe("Dashboard", () => {
...
+  ip: '127.0.0.1',
+  isConfigured: true,
+  port: '5555'
+}
+
+ const posArray_json = [
```
Comment (manoj22-k) [inline src/mtool/ui/src/containers/Dashboard/index.test.js:121]: please use camelCase for variables in js files

File: src/mtool/ui/src/containers/Dashboard/index.test.js [hunk 4]
```diff
@@ -676,7 +716,16 @@ describe("Dashboard", () => {
       "warnings": 0
     }
   ];
-
+  const hardware_json = {
```
Comment (manoj22-k) [inline src/mtool/ui/src/containers/Dashboard/index.test.js:719]: use camecase please

File: src/mtool/ui/src/containers/IbofOsOperations/Subsystem/index.test.js [hunk 5]
```diff
@@ -56,6 +56,24 @@ import i18n from "../../../i18n";
 
 jest.unmock("axios");
 
+
+// duplicate code 
+const listner_json = {
```
Comment (manoj22-k) [inline src/mtool/ui/src/containers/IbofOsOperations/Subsystem/index.test.js:61]: use camelcase for all varibales in js files

**Review by aswinssir — APPROVED** [review]
LGTM

**Review by manoj22-k — APPROVED** [review]
LGTM

**Pull request merged**